CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
-        public Guid CompanyId { get; set; }
+public decimal Salary { get; set; }
         public Company? Company { get; set; }
```

Comment:
Avoid leaving TODO comments and debug/test fields in production code. The 'salary' field appears redundant, is untyped (string instead of decimal), and does not follow naming conventions. This should be removed before merging.

```suggestion
// public string salary;  // Employee compensation - stores amount
```